src/DataTypeFactory.php

```diff
@@ -27,23 +19,19 @@ class DataTypeFactory {
 	private $types = array();
 
 	/**
-	 * Maps type id to a DataType builder spec.
-	 * See buildType() for more information.
-	 *
-	 * @var callable[]
+	 * @var string[]
 	 */
-	private $typeBuilders = array();
+	private $valueTypes = array();
 
 	/**
-	 * @since 0.1
+	 * @since 0.5
 	 *
-	 * @param callable[] $typeBuilders An array mapping type IDs to type builders. A type builder
-	 *        is a callable that takes a type ID as a parameter and returns a DataType object.
-	 *        Alternatively, a DataType object may be provided directly.
+	 * @param string[] $valueTypes
 	 */
-	public function __construct( array $typeBuilders = array() ) {
-		//TODO: check element type
-		$this->typeBuilders = $typeBuilders;
+	public function __construct( array $valueTypes ) {
```

Comment:
The constructor parameter not being optional was a breaking change, not updated correctly everywhere. See #51.